src/binary/create-logger.ts

```diff
@@ -92,3 +96,22 @@ export function createLogger(flags: LoggerFlags) {
 
   return logger;
 }
+
+// Heavily based on https://github.com/winstonjs/logform/blob/master/simple.js
+function simpleFormat() {
+  return winston.format((info: winston.Logform.TransformableInfo) => {
+    let stringifiedRest = jsonStringify({
+      ...info,
+      level: undefined,
+      message: undefined,
+      label: undefined,
+    });
+    stringifiedRest = stringifiedRest !== '{}' ? ` ${stringifiedRest}` : '';
+
+    const label = info.label ? ` [${info.label}]` : '';
+
+    info[MESSAGE] = `${info.level}:${label} ${info.message}${stringifiedRest}`;
+
+    return info;
+  })();
+}
```

Comment:
This is a custom format based on [winston.format.simple](https://github.com/winstonjs/logform/blob/master/simple.js) with an extra, optional `label`.